rep-0153.rst

```diff
@@ -0,0 +1,118 @@
+REP: 153
+Title: ROS distribution files
+Author: Dirk Thomas <dthomas@osrfoundation.org>
+Status: Final
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 27-Oct-2018
+Post-History: 27-Oct-2018
+
+
+.. contents::
+
+Abstract
+========
+This REP updates the specification of the ROS distribution files facilitated in
+the building, packaging, testing and documenting process.
+
+The intention is to annotate a ROS distribution with more information (like the
+"major" version (``1`` or``2``) to avoid the need to externally define and
+update these metadata.
```

Comment:
Do we need to update this intent sentence now?

Reply:
I am not sure what is unclear / not up-to-date atm. Please suggest an update.

Reply:
The example in the abstract still seems to belong more to the original proposal than the updated one. Since the ROS "major" version is described by the `distribution_type` key but the value we intend to use is not `1` or `2` but `ros1` or `ros2`.

Can't use the suggestion GUI for a multiline change.

```
The intention is to annotate a ROS distribution with more information,
such as whether the distribution should use ROS 1 or ROS 2 semantics,
to avoid the need to externally define and update these metadata.
```

Reply:
Sounds good to me: d7c643e577b2dacde73c82ad97bd0c6f4aa66002